content/webapp/components/IIIFItem/index.tsx

```diff
@@ -269,9 +273,10 @@ const IIIFItem: FunctionComponent<ItemProps> = ({
           setImageContainerRect={setImageContainerRect}
         />
       );
+
     case ((item.type === 'Sound' && !exclude.includes('Sound')) ||
       (item.type === 'Audio' && !exclude.includes('Audio'))) &&
-      Boolean(item.id):
+      !!item.id:
```

Comment:
OK I don't know _why_ but this change makes TS happy later when we use `item.id` as its satisfied that it's not undefined, whereas `Boolean()` did not fill that need 🤷‍♀️